tests/test_internals/test_roi_extractors_module.py

```diff
@@ -862,6 +862,8 @@ def test_default_iterator(self):
         # NWB stores images as num_columns x num_rows
         expected_two_photon_series_shape = (self.num_frames, self.num_columns, self.num_rows)
         assert two_photon_series_extracted.shape == expected_two_photon_series_shape
+        expected_two_photon_series_data = self.imaging_extractor.get_video().transpose((0, 2, 1))
+        assert_array_almost_equal(two_photon_series_extracted, expected_two_photon_series_data)
```

Comment:
following the discussion [here](https://github.com/catalystneuro/roiextractors/issues/193#issuecomment-1213338165) I wasn't sure whether to use strong equality test, so I chose to be consistent with [ecephys test](https://github.com/catalystneuro/neuroconv/blob/27357f8145746f2279e580b8a5565eaa26a98f96/tests/test_internals/test_si.py#L427)

Reply:
I think the argument of @CodyCBakerPhD  is stronger here. There is no reason why the float representation of this object should be different so maybe you want to know if they do change.

The argument over there in the linked thread is because `check_segmentation_equal` is used to compare objects that come from "farther away" (such as the use we do in our gin tests to compare the extractors with their versions from nwb or to compare the extractor with is marshaled version). Over there, in `check_segmentations_equal` I think we might want to have the two tests and allow the stricter version to be skipped in some cases.